octoprint_mrbeam/static/js/laser_job_done_viewmodel.js

```diff
@@ -57,6 +57,23 @@ $(function () {
             self.is_job_done(false);
             self.job_duration(0);
             self._fromData(payload);
+
+            // For styling purposes
+            let materialBoxElement1 = $(".modal__material-details-box--1");
+            let materialBoxElement2 = $(".modal__material-details-box--2");
+            let materialBoxElement3 = $(".modal__material-details-box--3");
+            if(materialBoxElement2.length >= 0){
```

Comment:
what are we doing here? this seems quite tricky to understand 🤔

Reply:
replaced by a for loop in Jinja